src/algorithms/expected_consensus.md

```diff
@@ -292,10 +339,16 @@ w[r+1] = w[r] + floor(1000 * (wForkFactor(wPowerFactor[r+1] + wBlocksFactor[r+1]
 
 with, for a given tipset ts in round r+1:
 
-- wBlocksFactor[r+1] = v * |blocksInTipset(ts)|
+- wBlocksFactor[r+1] = v * |blocksInTipset(ts)| = v * k
   - with v = vt * log2(totalPowerAtTipset(ts) * vs)/vs
 - wPowerFactor[r+1]  = log2(totalPowerAtTipset(ts))
-- wForkFactor[r+1]   = CDF(X, k) with X -> Bin(eNumberOfBlocksPerRound * numberOfMinersInPowerTable, 1/numberOfMinersInPowerTable)
+Take X -> Bin(eNumberOfBlocksPerRound * numberOfMinersInPowerTable, 1/numberOfMinersInPowerTable), for simplicity, we take:
+X -> Bin(eNumberOfBlocksPerRound * 10,000, 1/10,000). We have:
+- wForkFactor[r+1]   = 1 if |blocksInTipset(ts)| > E[X] - 2*stdDev[X], CDF(X, |blocksInTipset(ts)) otherwise.
```

Comment:
Define `E[X]` as a const for the network.

----

If I understand correctly `wForkFactor` is a fractional multiplier (`wForkFactor <= 1`) for the other factors (notation of `w[r+1]` is not clear in this regard.
Then the value should be:
1. Split into numerator and denominator.
2. Well defined for values of `|blocksInTipset(ts)|`.

Given its current spec, it is impossible for two different people to arrive at the same values.

Reply:
This is great feedback. Thanks @kubuxu for the review. Super useful.